streampipes-extensions/streampipes-connectors-influx/src/main/java/org/apache/streampipes/extensions/connectors/influx/sink/InfluxDbClient.java

```diff
@@ -67,7 +70,7 @@ private void connect() throws SpRuntimeException {
     var databaseName = connectionSettings.getDatabaseName();
 
     // Checking whether the database exists
-    if (!InfluxRequests.databaseExists(influxDb, databaseName)) {
+    if (!influxClientProvider.databaseExists(influxDb, databaseName)) {
       LOG.info("Database '" + databaseName + "' not found. Gets created ...");
       createDatabase(databaseName);
```

Comment:
Can we use `createDatabase()` from the InfluxClientProvider here?

Reply:
No, this only makes a check if the database does exist